yaksh/documentation/moderator_docs/creating_course.rst

```diff
@@ -37,21 +40,81 @@ Features in Courses
 
      The following features are available for courses
 
-        * Course Name
+        * **Course Name**
             Clicking on course name link will display all the enrolled, rejected and requested students list. Moderator can accept or reject the student.
-        * Quiz Name
-            Clicking on the quiz name will let you edit the quiz.
-        * Question Paper
-            Click on the **Add** link to create a Question Paper for associated Quiz.
-            If a question paper is already created, click on the Question Paper link to edit question paper.
-        * Add Teacher
+        * **Module Name**
+            Click to edit a module added to the course
+        * **Lesson or Quiz Name**
+            Click to edit a Lesson or Quiz added to the course
+
+            In edit quiz you can also attempt the quiz in two modes - 
+                * **God Mode** - In God mode you can attempt quiz without any time or eligibilty constraints.
+                * **User Mode** - In user mode you can attempt quiz the way normal users will attempt i.e.
+                    
+                    * Quiz will have the same duration as that of the original quiz.
+                    * Quiz won't start if the course is inactive or the quiz time has expired.
+                    * You will be notified about quiz prerequisites.(You can still attempt the quiz though)
```

Comment:
You will be notified about quiz prerequisites.(You can still attempt the quiz though)**.** (End it with a full stop)